docs/spark-standalone.md

```diff
@@ -89,6 +89,7 @@ Once you've set up this file, you can launch or stop your cluster with the follo
 - `sbin/start-slave.sh` - Starts a slave instance on the machine the script is executed on.
 - `sbin/start-all.sh` - Starts both a master and a number of slaves as described above.
 - `sbin/stop-master.sh` - Stops the master that was started via the `sbin/start-master.sh` script.
+- `sbin/stop-slave.sh` - Stops all slave instances on the machine the script is executed on.
```

Comment:
PS, while you're here, can you replace 'slave' with 'worker' in these descriptions? It's not so much because of, well, the unfortunate naming of these scripts, but because the scripts and rest of the docs actually refer to 'workers' consistently. conf/slaves and the script name doesn't change, but we can say that they control 'worker instances'